src/js/components/search/SearchSidebar.jsx

```diff
@@ -25,31 +26,81 @@ import PSCCheckboxTreeContainer from 'containers/search/filters/psc/PSCCheckboxT
 import PricingTypeContainer from 'containers/search/filters/PricingTypeContainer';
 import SetAsideContainer from 'containers/search/filters/SetAsideContainer';
 import ExtentCompetedContainer from 'containers/search/filters/ExtentCompetedContainer';
+import DEFCheckboxTree, { NewBadge } from 'containers/search/filters/def/DEFCheckboxTree';
 
-import KeywordHover from 'components/search/filters/keyword/KeywordHover';
+import {
+    KeyWordTooltip,
+    withAdvancedSearchTooltip,
+    DEFTooltip
+} from 'components/search/filters/tooltips/AdvancedSearchTooltip';
 
 import { Filter as FilterIcon } from 'components/sharedComponents/icons/Icons';
 import FilterSidebar from 'components/sharedComponents/filterSidebar/FilterSidebar';
 import * as SidebarHelper from 'helpers/sidebarHelper';
 
 const filters = {
     options: [
-        'Keyword',
-        'Time Period',
-        'Award Type',
-        'Agency',
-        'Treasury Account Symbol (TAS)',
-        'Location',
-        'Recipient',
-        'Recipient Type',
-        'Award Amount',
-        'Award ID',
-        'CFDA Program',
-        'North American Industry Classification System (NAICS)',
-        'Product or Service Code (PSC)',
-        'Type of Contract Pricing',
-        'Type of Set Aside',
-        'Extent Competed'
+        {
+            title: 'Keyword',
+            tooltip: withAdvancedSearchTooltip({
+                icon: "info",
+                tooltipComponent: <KeyWordTooltip />
+            })
+        },
+        {
+            title: 'Time Period'
+        },
+        {
+            title: 'Award Type'
+        },
+        {
+            title: 'Agency'
+        },
+        {
+            title: 'Treasury Account Symbol (TAS)'
+        },
+        {
+            title: 'Location'
+        },
+        {
+            title: 'Recipient'
+        },
+        {
+            title: 'Recipient Type'
+        },
+        {
+            title: 'Award Amount'
+        },
+        {
+            title: 'Award ID'
+        },
+        {
+            title: 'Disaster and Emergency Fund (DEF) Code',
```

Comment:
this should be after CFDA Program according to the mock

Reply:
@MaxwellKendall is this not the case? Did the mock change? Please explain why this conversation was resolved?

Reply:
@jonhill13 https://github.com/fedspendingtransparency/usaspending-website/pull/1709/commits/4b31be482c7250a57aadbd36b5fb6eb989b2aebb